### PR TITLE
docs: correct the formatting

### PR DESCRIPTION
**Pull request by santoshyadavdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers, public-api

File: CHANGELOG.md [hunk 1]
```diff
@@ -397,9 +393,7 @@ then your code will not longer compile. If you need to mutate the array, you
 should now take a copy (e.g. by calling `slice()`) and mutate the copy.
 * **common:** When passing a date-time formatted string to the `DatePipe` in a format that contains
 fractions of a millisecond, the milliseconds will now always be rounded down rather than
-to the nearest millisecond.
-
-Most applications will not be affected by this change. If this is not the desired behaviour
+to the nearest millisecond.Most applications will not be affected by this change. If this is not the desired behaviour
```
Comment (atscott) [inline CHANGELOG.md:396]:
```suggestion
to the nearest millisecond. Most applications will not be affected by this change. If this is not the desired behaviour
```

**Review by atscott — APPROVED** [review]
Reviewed-for: public-api

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: public-api

**Comment by AndrewKushnir** [comment]
FYI, adding the "cleanup" label based on the comment from Andrew: https://github.com/angular/angular/pull/39534#pullrequestreview-522792397, will remove it once the comment is addressed. Thank you.

**Review by IgorMinar — APPROVED** [review]
lgtm after @atscott's feedback is addressed. thank you!

**Comment by santoshyadavdev** [comment]
Sorry for the delay, it's done 👍

**Comment by AndrewKushnir** [comment]
Thanks for the update @santoshyadavdev! I've removed the "cleanup" label and added the "merge" one to add this PR to the merge queue.

Note: I've also updated the "target" label from "patch" to "rc" (so it will be merged into `11.0.x` and `master` branches), since the PR updates release notes for the `11.0.0-next.5` version which is not present in `10.2.x` (patch) branch.

**Pull request closed without merging**